pwa/src/pages/translation-tables/[table].tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import Layout from "../../components/common/layout";
+import TranslationTable from "../../components/translations/translationTable";
+import TranslationForm from "../../components/translations/translationForm";
+
+const IndexPage = (props) => {
+  const tableName: string = props.params.id === "new" ? null : props.params.table
```

Comment:
You're doing three checks within this file if `tableName !== "new"` **after this declaration**.

It can never be "new" right? Only if the table name itself is "new"? Because you're setting it to `null`.

Reply:
Another not: `props.params.id` doesn't exist at all here.